src/test/java/com/answerdigital/answerking/builder/CategoryTestBuilder.java

```diff
@@ -0,0 +1,77 @@
+package com.answerdigital.answerking.builder;
+
+import com.answerdigital.answerking.model.Category;
+import com.answerdigital.answerking.model.Product;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.answerdigital.answerking.util.DateTimeUtility.getDateTimeAsString;
+
+public class CategoryTestBuilder {
+
+    private Long id = 1L;
+    private String name;
+    private String description;
+    private String createdOn;
+    private String lastUpdated;
```

Comment:
Would it be OK to change these to LocalDateTime - just trying to future proof when (or if) we switch to the Hibernate timestamp annotations!